docs/development/packaging.md

```diff
@@ -1,27 +1,115 @@
-# Deb packaging HOWTO
+# Deb package for Ubuntu
 
+The way Ralph is made makes changing versions of its dependencies challenging
```

Comment:
This is a very long sentence, some punctuation would be helpful.

Reply:
Done